src/arch/forte_architecture_time.h

```diff
@@ -76,4 +77,6 @@ uint_fast64_t getNanoSecondsRealtime();
 
 uint_fast64_t trackFakeForteTime();
 
+std::string getRealtimeString();
```

Comment:
please move this declaration into `devlog.cpp` currently we only need it there.

Reply:
Done.